check-docstyle.sh

```diff
@@ -28,15 +28,15 @@ function prepare_venv() {
 
     printf "%sOK%s\n" "${GREEN}" "${NORMAL}" >&2
 
-    ${PYTHON} -m venv "venv" && source venv/bin/activate && pip install pydocstyle >&2
+    ${PYTHON} -m venv "venv" && source venv/bin/activate && python -m pip install pydocstyle >&2
```

Comment:
${PYTHON} -m pip ... ?

Reply:
After the `source` command it should not matter any more, or does it?